.circleci/config.yml

```diff
@@ -7,13 +7,13 @@ jobs:
   build:
     working_directory: ~/EGG
     docker:
-      - image: circleci/python:3.6
+      - image: cimg/python:3.12
```

Comment:
EGG was developed and tested with python ~3.6 maybe, why testing with 3.12?

Reply:
Isn't it time to update to a more recent version? Moreover, the docker circle image has been [deprecated](https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034/9)